modules/data.atmosphere/vignettes/tdm_downscaling.Rmd

```diff
@@ -14,24 +14,42 @@ Examples:
 ```{r}
 library(PEcAn.data.atmosphere)
 library(PEcAn.DB)
-download.Fluxnet2015(sitename = 'Niwot Ridge Forest/LTER NWT1 (US-NR1)', username = 'pecan', overwrite = FALSE, outfolder = '~/Example', 
-   start_date = '2000/01/01', end_date = '2014/12/31')
-
-bety     <- list(user = 'bety', password = 'bety', host = 'localhost', dbname = 'bety', driver = 'PostgreSQL', write = TRUE)
-bety$con <- db.open(bety) 
-format   <- query.format.vars(input.id = 5000000005, bety = bety)
-met2CF.csv(lat = 40.0329, lon = -105.546, overwrite = FALSE, in.path = '~/Example', in.prefix = 'FLX_US-NR1_FLUXNET2015_SUBSET_HH_1998-2014_1-3', outfolder = '~/Example/CF', start_date = '2000/01/01', end_date = '2014/12/31', format)
-
-metgapfill(lst = -7, overwrite = FALSE, in.path = '~/Example/CF', in.prefix = 'FLX_US-NR1_FLUXNET2015_SUBSET_HH_1998-2014_1-3', outfolder = '~/Example/gapfill', start_date = '2000/01/01', end_date = '2014/12/31')
-
-nc.merge(in.path = "~/Example/gapfill", in.prefix = "FLX_US-NR1_FLUXNET2015_SUBSET_HH_1998-2014_1-3", outfolder = "~/Example/NR1_training_data", start_date = "2000-01-01", end_date = "2014-12-31", upscale = "hour")
+sitename   <- 'Niwot Ridge Forest/LTER NWT1'
+outfolder  <- dir.create(file.path('~/Example'))
+start_date <- '2000/01/01'
+end_date   <- '2014/12/31'
+download.Fluxnet2015(username = 'pecan', overwrite = FALSE)
```

Comment:
@jsimkins2 here and below -- even when you define these variables, you still need to **explicitly pass them to the functions**!

Also, I think you misunderstood my suggestion about using file.path -- you're still hard coding the paths throughout this script rather than using file.path to make the **names** on the fly. Also, you don't need to keep calling dir.create on every single one of them because that'd done on the fly in the functions, it's just the root directory that you need to define and create on the fly.  Right now, you're also redefining variables all over the place (e.g. outfolder) which isn't what I intended.  Set outfolder to "~/Example" and then never redefine it. Just in the function calls themselves pass in file.path(outfolder,<subfolder>) to specify each subfolder.

Finally, I was suggesting that you define ALL constants up front -- it's much easier for users if those are set once at the top rather than having to hunt through the code to figure out all the things that have to be modified.  And if the meaning of a variable isn't obvious, please provide a comment defining what it is.